### PR TITLE
N28: Release

### PR DESCRIPTION
**Pull request by AngelOnFira**
Part of #852 

Netlify Preview (b874f85): https://agitated-saha-af60ca.netlify.app/

Remaining steps:

- [x] Add discussions links
- [x] Spelling and grammar pass
- [ ] Final review
- [ ] Publish
- [ ] Posts
- [ ] Comment links

### CODE REVIEW

**Comment by PsichiX** [comment]
Rust CUDA seems to have missing link in the title - `[Rust CUDA]`

**Review by 17cupsofcoffee — COMMENTED** [review]
Looks good, just some minor comments:

* The empty sections still need removing (and the corresponding TOC items), and the requests for contribution section needs filling in.
* Halloween Mahjong Solitaire has some repeated phrasing in the first line:

> Halloween Mahjong Solitaire (GitHub) **by Syn-Nine** is a game created **by Syn9** for Game Developers Refuge 4x4x4 Challenge, October 2021.

* Bendyworm has the dev's name inside the brackets instead of outside.
* Might it be worth making the 'Raytracer Challenge' image link to the video playlist?

**Comment by AngelOnFira** [comment]
@ozkriff @17cupsofcoffee I couldn't make a PR on this repo for the other news section, since I made this branch from my fork, so I just merged it and we can request any changes in this PR.

https://github.com/rust-gamedev/rust-gamedev.github.io/pull/887/commits/f9f781dbd3dfe78fa3ee6b135882be6e6e2667f9

File: content/news/028/index.md [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-by watching [the coordination issues][coordination].
-Want something mentioned in the next newsletter?
-[Send us a pull request][pr].
-Feel free to send PRs about your own projects!
+You can follow the newsletter creation process by watching [the coordination
+issues][coordination]. Want something mentioned in the next newsletter? [Send us
+a pull request][pr]. Feel free to send PRs about your own projects!
 
 [Rust]: https://rust-lang.org
 [join]: https://github.com/rust-gamedev/wg#join-the-fun
```
Comment (ozkriff) [inline content/news/028/index.md:22]: no `- [Other News](#other-news)` in the ToC

**Review by ozkriff — APPROVED** [review]

**Comment by AngelOnFira** [comment]
Going for the merge now, as I think I've more or less gotten an approval from enough parties :)

**Pull request merged**